src/ripple/protocol/LedgerFormats.h

```diff
@@ -243,6 +243,9 @@ enum LedgerSpecificFlags {
         0x10000000,               // True, reject new paychans
     lsfDisallowIncomingTrustline =
         0x20000000,               // True, reject new trustlines (only if no issued assets)
+    lsfAMM [[maybe_unused]] = 0x40000000, // True, AMM account
+    lsfAllowClawback =
+        0x80000000,               // True, enable clawback 
```

Comment:
how did you come up with this particular value `0x80000000` for this flag? I know that it needs to be different from the other flags and complementary to `NoFreeze` flag, but were there any other considerations?

Reply:
I chose it because this is the next available flag. I don't think there is a clean way of deciding the flag value. But what I did is I looked over the [amendment proposals](https://github.com/XRPLF/XRPL-Standards/discussions), as well as their PRs, to find out which flag values are in use. 

Since the AccountRoot `Flags` is a 32-bit bitmask, I chose `0x80000000` because it is simply the last remaining flag value that haven't been used.